qiita_pet/handlers/download.py

```diff
@@ -77,6 +46,149 @@ def _generate_files(self, header_name, accessions, filename):
         self.write(text)
         self.finish()
 
+    def _list_dir_files_nginx(self, dirpath):
+        """Generates a nginx list of files in the given dirpath for nginx
+
+        Parameters
+        ----------
+        dirpath : str
+            Path to the directory
+
+        Returns
+        -------
+        list of (str, str, str)
+            The path information needed by nginx for each file in the
+            directory
+        """
+        basedir = get_db_files_base_dir()
+        basedir_len = len(basedir) + 1
```

Comment:
Is this, and its use below, basically the same as `os.dirname`? Same comment for line 89 and its use in the next function

Reply:
No, this is used to remove a prefix from the path that can be formed by multiple dirs.